CMakeLists.txt

```diff
@@ -118,7 +118,6 @@ set(VCLIBS_DEBUG_OPTIONS "$<$<COMPILE_LANGUAGE:CXX>:/Od>")
 set(VCLIBS_RELEASE_OPTIONS "$<$<COMPILE_LANGUAGE:CXX>:/O2;/Os>")
 
 if(BUILD_TESTING)
-    enable_testing()
     add_subdirectory(tests)
 endif()

```

Comment:
No change requested:

> We still use `BUILD_TESTING` to control including the test targets in the generated build system, but I'm not certain there's a good reason to do so. Maybe to enable folks without python to build but not test the STL?

Keeping the `BUILD_TESTING` option seems like a good idea for that reason. The Python requirement is indeed limited to the `tests` subdirectory:

https://github.com/microsoft/STL/blob/89ca073a866d24db42b3f4c5e406be4b28a148f4/tests/CMakeLists.txt#L26